ai-validator/libs/tests/test-1.script.md

```diff
@@ -0,0 +1,117 @@
+## Steps
+
+### 1. Click 'Service Level Objectives (SLO)'
+
+In the left panel, under Application Signals, click 'Service Level Objectives (SLO)'.
+
+### 2. Search for 'appointment service availability'
+
+In the search field with placeholder text 'Filter SLOs by text, property or value', type 'appointment service availability' and press Enter.
+
+### 3. Click 'appointment-service-get/FunctionHandler'
+
+Click the hyperlink 'appointment-service-get/FunctionHandler' in the 'Service Level Objectives (SLO)' list in the main panel.
+
+### 4. Click 'Service operations'
+
+Click the 'Service operations' button.
+
+### 5. Access the graph and open the popup.
+
+Access the graph and open the popup, PASS in 2 and 6 as a PARAMETERS.
+
+**Constraints:**
```

Comment:
Is this `Constraints` recommended by broweruse? Maybe document relevant information in the readme